src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -26,29 +26,15 @@ export PAI_DEFAULT_FS_URI={{{ hdfsUri }}}
 exec 13>$LAUNCHER_LOG_DIR/YarnContainerDebug.log
 BASH_XTRACEFD=13
 
-function debug_log()
-{
-  if [ $# -gt 1 ]; then
-    local prefix=$1
-    local message=$2
-    echo "[DEBUG] $prefix: $message"
-  fi
-}
 
 function exit_handler()
 {
   rc=$?
-  local handler="Yarn container exit handler"
-  debug_log "$handler"  "EXIT signal received in yarn container, performing clean up action..."
+  printf "[DEBUG] EXIT signal received in yarn container, performing clean up action...\n"
 
-  debug_log "$handler" "write exit code to file"
-  debug_log "$handler" "yarn container exit code: $rc"
-  debug_log "$handler" "exit code file path: /var/lib/hadoopdata/nm-local-dir/nmPrivate/$APP_ID/$CONTAINER_ID/$CONTAINER_ID.pid.exitcode"
+  printf "[DEBUG] Write exit code $rc to file /var/lib/hadoopdata/nm-local-dir/nmPrivate/$APP_ID/$CONTAINER_ID/$CONTAINER_ID.pid.exitcode.\n"
   echo $rc > "/var/lib/hadoopdata/nm-local-dir/nmPrivate/$APP_ID/$CONTAINER_ID/$CONTAINER_ID.pid.exitcode"
 
-  debug_log "$handler" "clean the container code"
-  rm -fr tmp/pai-root/code 2>/dev/null
```

Comment:
We are not going to delete the code? Will Yarn clean it up?

Reply:
Yes, all folders under yarn local directory will be cleaned.